packages/odyssey-react/src/components/Banner/index.tsx

```diff
@@ -82,13 +83,13 @@ export type Props = ComponentProps & DismissableComponentProps;
     ...rest
   } = props;
 
+  const variantClass = `variant${variant[0].toUpperCase()}${variant.slice(1)}`
```

Comment:
I did share this snippet in @edburyenegren-okta 's PR recently, but I don't love the run time cost for manipulating strings we control just to create style classes. Is there strong opposition to simplifying the CSS class names to match and allowing us less code in the component here?